src/drivers/uavcan/actuators/esc.cpp

```diff
@@ -75,7 +76,7 @@ UavcanEscController::update_outputs(bool stop_motors, uint16_t outputs[MAX_ACTUA
 	 */
 	const auto timestamp = _node.getMonotonicTime();
 
-	if ((timestamp - _prev_cmd_pub).toUSec() < (1000000 / MAX_RATE_HZ)) {
+	if (_uavcan_use_rate_limit == 1 && (timestamp - _prev_cmd_pub).toUSec() < (1000000 / MAX_RATE_HZ)) {
```

Comment:
Should the rate limit parameter actually have the value of MAX_RATE_HZ, instead of just 0/1? Then you could variate it and not limit to hard-coded macro?

Reply:
I will continue to use a param to toggle the use of rate limit for ESC command publish here. I want to keep the rate limit comparison logic as close to the original UAVCAN implementation as possible. This is due to realization after the flight test that rate limit here is not needed or actually a duplication (which weirdly leads to very slow cmd rate << MAX_RATE_HZ) as it's already done in https://github.com/tiiuae/px4-firmware/blob/ff436d4aa7b471af3471f4e73d836242a12da9ce/src/drivers/uavcan/actuators/hardpoint.cpp#L81 to set the publish rate